src/applications/vaos/utils/appointment.js

```diff
@@ -397,7 +407,7 @@ function getInstructions(appointment) {
   return null;
 }
 
-function getAppointmentStatus(appointment) {
+export function getAppointmentStatus(appointment, isPastAppointment) {
```

Comment:
What do you think about keeping this function internal? I want to make sure it doesn't spread throughout the app again. It looks like you exported this for testing. You could instead do what the `transformAppointment` tests do, which is group tests for individual properties together.